src/decluttering.md

```diff
@@ -0,0 +1,24 @@
+# Decluttring the UI and reducing cognitive load
```

Comment:
I kinda don't like the _Decluttring_ word here. At first I didn't understand what it meant and also that the sentence sounds weird. Would it be possible to improve it? I saw that Kia mentioned `Reducing UI clutter` but you like this better?

Reply:
We could say "Reducing UI clutter and user's cognitive load"

Reply:
Decluttring is a typo. It should be decluttering. But I'm absolutely fine with rephrasing to avoid the word 🙂

Reply:
Pushed a fix with the heading "Reducing UI clutter and cognitive load".